packages/botonic-nlu/src/file-utils.js

```diff
@@ -0,0 +1,114 @@
+import fs from 'fs'
+import path from 'path'
+import colors from 'colors'
+import { parseUtterance } from './preprocessing'
+import { filterObjectByWhitelist } from './utils'
+import {
+  NLU_CONFIG_FILENAME,
+  UTTERANCES_DIRNAME,
+  MODELS_DIRNAME,
+  NLU_DATA_FILENAME
+} from './constants'
+
+const FILE_OPEN_EXCEPTION = error => colors.red(`${error}`)
+const CONFIG_NOT_FOUND_EXCEPTION = flagLang =>
+  colors.red(
+    `No configuration found for '${flagLang}' in ${NLU_CONFIG_FILENAME}.`
+  )
+
+export function readDir(dirPath) {
+  try {
+    return fs.readdirSync(dirPath).filter(dirName => dirName !== '.DS_Store')
+  } catch (e) {
+    throw FILE_OPEN_EXCEPTION(e)
+  }
+}
+
+export function readFile(filePath, fileEncoding = 'utf-8') {
+  try {
+    return fs.readFileSync(filePath, fileEncoding)
+  } catch (e) {
+    throw FILE_OPEN_EXCEPTION(e)
+  }
+}
+
+export function readJSON(jsonPath) {
+  return JSON.parse(readFile(jsonPath))
+}
+export function writeJSON(jsonPath, object) {
+  fs.writeFileSync(jsonPath, JSON.stringify(object, null, 2))
+}
+
+export function createDir(path) {
+  fs.mkdirSync(path)
+}
+
+export function pathExists(path) {
+  return fs.existsSync(path)
+}
+
+export function appendNewLine(path, str) {
+  fs.appendFileSync(path, `\n${str}`)
+}
+
+export function getIntentName(fileName) {
+  try {
+    return fileName.match(/(.*).txt/)[1]
+  } catch (e) {
+    console.log(
+      `${fileName} is not a valid. File must be of type IntentName.txt`
+    )
+  }
+}
+
+export function loadDevData(nluPath, languages) {
```

Comment:
is "devData" the best name for this?

Reply:
will you leave it as "loadNLUData" maybe?

Reply:
whatever name you choose, but try to stick to it. for instance on this method you call it config. 
Maybe you can think about it and do it on a separate PR